collector/sysctl_bsd.go

```diff
@@ -108,6 +109,29 @@ func (b bsdSysctl) Value() (float64, error) {
 		// This conversion maintains the usec precision.  Using
 		// the time package did not.
 		tmpf64 = unix + (usec / float64(1000*1000))
+	case bsdSysctlTypeCLong:
+		raw, err := unix.SysctlRaw(b.mib)
+		if err != nil {
+			return 0, err
+		}
+
+		if len(raw) != (C.sizeof_long) {
+			if len(raw) != (C.sizeof_int) {
+				return 0, fmt.Errorf(
+					"length of bytes received from sysctl (%d) does not match expected bytes (%d)",
+					len(raw),
+					C.sizeof_long,
+				)
+			}
+
+			// Not sure this is valid for all CLongs.  It is at
+			// least for vfs.bufspace:
+			//   https://github.com/freebsd/freebsd/blob/releng/10.3/sys/kern/vfs_bio.c#L338
+			tmpf64 = float64(*(*C.int)(unsafe.Pointer(&raw[0])))
+			break
```

Comment:
Unneeded.

Reply:
The `break` at 131 is what stops 134 for overwriting - at least that's my intention.

Reply:
Apologies, may have misread the diff.

Reply:
Still valid. ;)